### ISSUE TITLE
Include JavaDoc comments for JAX-RS REST methods

### ISSUE THREAD

**Issue by Q-Man** [vojtechhabarta/typescript-generator#423]
It would be great if the JavaDoc comments of the JAX-RS REST methods would be integrated in the generated typescript methods with the following configuration:

```
<generateJaxrsApplicationClient>true<generateJaxrsApplicationClient>
<javadocXmlFiles>true</javadocXmlFiles>
```
At the moment there's just a general comment added to the generated methods like:

    /**
     * HTTP GET /foo/{id}
     * Java method: com.foo.FooResource.getFoo
     */

The comments, @param and @return descriptions of the JavaDoc could be a great help.

**Comment by vojtechhabarta** [comment]
I committed (4a5a4d29fb857831d46c162375527e1dbf40e2d9) Javadoc for REST methods.
It would be great if you could check it.

Please note that `javadocXmlFiles` parameter is list of files with Javadoc exported using xml-doclet.
See https://github.com/vojtechhabarta/typescript-generator/wiki/Javadoc.

**Comment by Q-Man** [comment]
I'm using the typescript generator as maven plugin, but the updated version 2.17.563 is not public yet. Do you have a tip how to test it?

**Comment by vojtechhabarta** [comment]
You can clone this Git repository and build it locally or you can just wait for the released version.

**Comment by vojtechhabarta** [comment]
Released in version [2.18.565](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.18.565).

**Comment by Q-Man** [comment]
> I committed ([4a5a4d2](https://github.com/vojtechhabarta/typescript-generator/commit/4a5a4d29fb857831d46c162375527e1dbf40e2d9)) Javadoc for REST methods.
> It would be great if you could check it.
> 
> Please note that `javadocXmlFiles` parameter is list of files with Javadoc exported using xml-doclet.
> See https://github.com/vojtechhabarta/typescript-generator/wiki/Javadoc.

It's working, thank you!
